src/main/java/bio/terra/flight/study/delete/StudyDeleteFlight.java

```diff
@@ -0,0 +1,19 @@
+package bio.terra.flight.study.delete;
+
+import bio.terra.dao.StudyDao;
+import bio.terra.stairway.Flight;
+import bio.terra.stairway.FlightMap;
+import org.springframework.context.ApplicationContext;
+
+public class StudyDeleteFlight extends Flight {
+
+    public StudyDeleteFlight(FlightMap inputParameters, Object applicationContext) {
+        super(inputParameters, applicationContext);
+
+        // get the required daos to pass into the steps
+        ApplicationContext appContext = (ApplicationContext) applicationContext;
+        StudyDao studyDao = (StudyDao)appContext.getBean("studyDao");
+
+        addStep(new DeleteStudyMetadataStep(studyDao));
```

Comment:
We will need to add a delete step for the primary data. The deleteStudy method is implemented in the pdao. If you decide to do it in this PR, you can refer to my dataset flights PR.